Multithreading modified slides

aparna951992 opened the pull request:
@coderhs @manusajith 
Added modification to the slide presented during session along with explanation for the questions asked.

Commit:
Multithreading modified slides

coderhs commented:
@aparna951992 can you make it into two commits. Like the original slide and the modification you made, I can't make heads or tails out of this single commit.

aparna951992 commented:
1. Added how we can use multithreaded ruby program with an example of such a program.
2. Added the correct functionality of Thread.abort_on_exception.
3. Added details regarding how threads can be prioritized.
4. Added details regarding implementing thread exclusion.
5. Explained how multiprogramming is implemented as well as multithreading is implemented using examples.
6. Explained actors and fibres in detail as well as the problem while using fibre.
